src/style.css

```diff
@@ -71,12 +71,33 @@ h1 {
   font-size: 0.85rem;
 }
 
+.author:focus {
+  outline: 1px solid;
+  border-radius: 4px;
+}
+
```

Comment:
- [ ] The font of the text should match with the design. Please make sure you follow the design.